ui/src/main/java/edu/wpi/grip/ui/preview/TextAreaSocketPreviewView.java

```diff
@@ -35,7 +35,7 @@ public void onSocketChanged(SocketChangedEvent event) {
         final Socket socket = event.getSocket();
 
         if (socket == this.getSocket()) {
-            this.text.setText(socket.getValue().toString());
+            this.text.setText(socket.getValue().get().toString());
```

Comment:
This should better handle empty sockets, even if we think that won't ever happen.